packages/api/src/graphql/mutation.private.ts

```diff
@@ -471,13 +472,47 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
       args: {
         input: {type: GraphQLNonNull(GraphQLSubscriptionInput)}
       },
-      resolve(root, {input}, {authenticate, dbAdapter}) {
+      async resolve(root, {input}, {authenticate, dbAdapter}) {
         const {roles} = authenticate()
         authorise(CanCreateSubscription, roles)
 
         if (isTempUser(input.userID)) throw new Error('Can not update subscription with tempUser')
 
-        return dbAdapter.subscription.createSubscription({input})
+        const subscription = await dbAdapter.subscription.createSubscription({input})
+        if (!subscription) throw new Error('Subscription not created.')
+
+        // create invoice
```

Comment:
This logic is already handled in `memberContext` and i recommend to leave it there. `createSubscription` should not automatically create invoices. For that we have jobs in the `memberContext`. If you need an inovice i suggest to call/create another mutation.

Reply:
Will have a look at it.

Reply:
now I saw your intention and changed that.